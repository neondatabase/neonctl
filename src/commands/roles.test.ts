import { describe } from 'vitest';

import { test } from '../test_utils/fixtures';

describe('roles', () => {
  test('list', async ({ testCliCommand }) => {
    await testCliCommand([
      'roles',
      'list',
      '--project-id',
      'test',
      '--branch',
      'test_branch',
    ]);
  });

  test('create', async ({ testCliCommand }) => {
    await testCliCommand([
      'roles',
      'create',
      '--project-id',
      'test',
      '--branch',
      'test_branch',
      '--name',
      'test_role',
    ]);
  });

  test('delete', async ({ testCliCommand }) => {
    await testCliCommand([
      'roles',
      'delete',
      'test_role',
      '--project-id',
      'test',
      '--branch',
      'test_branch',
    ]);
  });
});
