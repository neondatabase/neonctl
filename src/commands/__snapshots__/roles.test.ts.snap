// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`roles > create 1`] = `
"name: test_role
created_at: 2019-01-01T00:00:00.000Z
"
`;

exports[`roles > delete 1`] = `
"name: test_role
created_at: 2016-11-16T15:39:18.000Z
"
`;

exports[`roles > list 1`] = `
"- name: test_role
  created_at: 2016-01-01T00:00:00Z
"
`;
